pkg/kubectl/cmd/expose.go

```diff
@@ -149,9 +149,9 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 	// For objects that need a pod selector, derive it from the exposed object in case a user
 	// didn't explicitly specify one via --selector
 	if s, found := params["selector"]; found && kubectl.IsZero(s) {
-		s, err := f.PodSelectorForObject(inputObject)
```

Comment:
@janetkuo yes, added the support in this PR - https://github.com/kubernetes/kubernetes/pull/20928. PTAL.